src/components/ReleaseService/ReleasePlan/ReleasePlanListRow.tsx

```diff
@@ -3,6 +3,8 @@ import { Link } from 'react-router-dom';
 import { Label, capitalize } from '@patternfly/react-core';
 import { CheckCircleIcon } from '@patternfly/react-icons/dist/esm/icons/check-circle-icon';
 import { ExclamationCircleIcon } from '@patternfly/react-icons/dist/esm/icons/exclamation-circle-icon';
+import { getApplicationDisplayName } from '~/components/Applications/application-utils';
```

Comment:
can we move this to common utils as this might be used across components.

Reply:
moved